library/src/scala/util/control/ControlException.scala

```diff
@@ -0,0 +1,18 @@
+package scala.util.control
+
+/** A parent class for throwable objects intended for flow control.
+ *
+ *  Instances of ControlException don't record a stacktrace and are therefore
+ *  much more efficient to throw than normal exceptions.
+ *
+ *  Unlike `ControlThrowable`, `ControlException` is a regular `RuntimeException`
+ *  that is supposed to be handled like any other exception.
+ *
+ *  Instances of `ControlException` should not normally have a cause.
+ *  Legacy subclasses may set a cause using `initCause`.
+ */
+abstract class ControlException(message: String | Null) extends RuntimeException(
```

Comment:
I thought that we wanted to make `Break` extend `Throwable` to avoid catching it in a catch that expects an `Exception`. Why did this change? What is the use case?

Reply:
We had a long discussion on contributors about that.